tests/acceptance/run.sh

```diff
@@ -517,9 +514,12 @@ fi
 # If the caller did not mention specific tags, skip the skipped tests by default
 if [ "${BEHAT_TAGS_OPTION_FOUND}" = false ]
 then
+	if [[ -z $BEHAT_FILTER_TAGS ]]
+	then
+		BEHAT_FILTER_TAGS="~@skip"
 	# If the caller has already specified specifically to run "@skip" scenarios
 	# then do not append "not @skip"
-	if [[ ! ${BEHAT_FILTER_TAGS} =~ "&&@skip&&" ]]
+	elif [[ ! ${BEHAT_FILTER_TAGS} =~ "@skip" ]]
```

Comment:
Yeah, what I want to explain is that if there are tags like this `@tag1&&@tag2&&@skipOnV2&&@tag3` then the condition `${BEHAT_FILTER_TAGS} =~ "@skip"` will be true and results `false` (due to `!`). But in this case, the condition should be `false` and should result to `true` so that the final tags can be `@tag1&&@tag2&&@skipOnV2&&@tag3&&~@skip`

Reply:
Try with this one `(^|&)@skip(&|$)`